sebsync.py

```diff
@@ -369,8 +436,23 @@ def sebsync(**kwargs):
     # --quiet wins over --verbose
     options.verbose = options.verbose and not options.quiet
 
+    # Use a local cache for storing ebook metadata
+    cache = {"epub": {}, "kindle": {}}
+    cachefile = Path("local.cache")
+    if cachefile.exists() and cachefile.is_file():
+        with open(cachefile, "rb") as f:
+            cache = pickle.load(f)
+
+    # Get the cache for the specific type of file we're syncing
+    if options.type == "kindle":
+        local_cache = cache["kindle"]
+    else:
+        local_cache = cache["epub"]
```

Comment:
I suggest for now that the cache (it still seems more like an "index" file to me, though it does seem fitting to store it in the platform cache directory) be only for Kindle books, as I don't see any utility in storing this information for EPUBs.

Reply:
I figured that it'd be easier to just treat both types of files the same rather than peppering a lot of `if options.type == "kindle": … else: …` bits throughout the code and tried to make robust, but I can see about having it only for Kindle files.